### PR TITLE
we have a new yacn server

### PR DESCRIPTION
**Pull request by etam**

### CODE REVIEW

File: core/serv/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The same binary acts as a command line interface for the service.
 | Data folder | `-d, --datadir <path>` | `YAGNA_DATADIR` | platform specific (see `--help`) | The folder in which the Daemon's SQL storage file is to be located | 
 | GSB URL | `-g, --gsb-url <url>` | `GSB_URL` | `tcp://127.0.0.1:7464` | Service Bus URL |
 | REST API URL | `-a, --api-url <url>` | `YAGNA_API_URL` | `http://127.0.0.1:7465` | Yagna REST API endpoints base URL |
-| Net Mk1 hub addr | N/A | `CENTRAL_NET_HOST` | `3.249.139.167:7464` | Centralized (Mk1 phase) Yagna network server address |
+| Net Mk1 hub addr | N/A | `CENTRAL_NET_HOST` | `yacn2.dev.golem.network:7464` | Centralized (Mk1 phase) Yagna network server address |
```
Comment (tworec) [inline core/serv/README.md:16]: this is not default value. Default is to get host and port from DNS

File: core/market/resolver/tests/sample/mod.rs [hunk 2]
```diff
@@ -92,7 +92,7 @@ pub static POC_DEMAND_PROPERTIES_JSON: &str = r#"
   "golem.node.debug.subnet": "piotr",
   "golem.node.id.name": "test1",
   "golem.srv.comp.expiration": 1590765503361,
-  "golem.srv.comp.task_package": "hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://3.249.139.167:8000/rust-wasi-tutorial.zip"
+  "golem.srv.comp.task_package": "hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://yacn2.dev.golem.network:8000/rust-wasi-tutorial.zip"
```
Comment (tworec) [inline core/market/resolver/tests/sample/mod.rs:95]: Pls do not change the Resolver module at this stage. We are frozen.
And it is only a string. Not interpreted by resolver nor any1. This is a test only.

File: agent/provider/readme.md [hunk 3]
```diff
@@ -409,12 +409,12 @@ cargo run -p ya-provider run
 ```
 
 ## Central setup
-We have centrally deployed (@ ip: `3.249.139.167`) three independent standalone modules/apps:
- - [net Mk1](https://github.com/golemfactory/yagna/blob/master/docs/net-api/net-mk1-hub.md) @ 3.249.139.167:7464 \
+We have centrally deployed (@ yacn2.dev.golem.network) three independent standalone modules/apps:
```
Comment (tworec) [inline agent/provider/readme.md:412]: great that you are updating the docs

File: core/serv/README.md (hunk 1 above)
Comment (tworec) [reply core/serv/README.md:16]: `.env-template` file also needs updating with same description as here + Wiki https://github.com/golemfactory/yagna/wiki/DotEnv-Configuration

**Review by tworec — CHANGES_REQUESTED** [review]

File: core/serv/README.md (hunk 1 above)
Comment (tworec) [reply core/serv/README.md:16]: I've removed the Wiki page. It makes no sense to have it tripled.

File: core/market/resolver/tests/sample/mod.rs (hunk 2 above)
Comment (etam) [reply core/market/resolver/tests/sample/mod.rs:95]: So maybe it should contain some intentionally unimportant host name like `12.34.56.78`?

Comment (tworec) [reply core/market/resolver/tests/sample/mod.rs:95]: From the resolver tests point of view, this is a bare string. You can put it `fusiek` :)
It does not need to be up to date. Its contents just mimics real values, that's all.

Comment (etam) [reply core/market/resolver/tests/sample/mod.rs:95]: Also, maybe in other files it's not important as well?

Comment (tworec) [reply core/market/resolver/tests/sample/mod.rs:95]: yes. thats my point. Let's do not spend any more time on that, as this is of far importance

File: core/serv/README.md (hunk 1 above)
Comment (etam) [reply core/serv/README.md]: updated

**Comment by etam** [comment]
I changed this PR target branch to master.

**Comment by etam** [comment]
@golemfactory/ya-integration @golemfactory/ya-payment I'd like to change hostnames in places where it's not actually resolved and used, to something that is obviously a placeholder, like `example.com` or `12.34.56,78`. Please tell which lines I changed I should change to placeholder.

File: core/market/resolver/tests/sample/mod.rs (hunk 2 above)
Comment (tworec) [inline core/market/resolver/tests/sample/mod.rs:95]:
```suggestion
  "golem.srv.comp.task_package": "hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://12.34.56.78:8000/rust-wasi-tutorial.zip"
```

File: core/market/resolver/tests/sample/mod.rs [hunk 4]
```diff
@@ -104,7 +104,7 @@ pub static POC_DEMAND_PROPERTIES_JSON_DEEP: &str = r#"
     },
     "srv.comp": {
       "expiration": 1590765503361,
-      "task_package": "hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://3.249.139.167:8000/rust-wasi-tutorial.zip"
+      "task_package": "hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://yacn2.dev.golem.network:8000/rust-wasi-tutorial.zip"
```
Comment (tworec) [inline core/market/resolver/tests/sample/mod.rs:107]:
```suggestion
      "task_package": "hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://12.34.56.78:8000/rust-wasi-tutorial.zip"
```

File: core/market/resolver/tests/sample/mod.rs [hunk 5]
```diff
@@ -113,7 +113,7 @@ pub static POC_DEMAND_PROPERTIES_FLAT: &'static [&'static str] = &[
     "golem.node.debug.subnet=\"piotr\"",
     "golem.node.id.name=\"test1\"",
     "golem.srv.comp.expiration=1590765503361",
-    "golem.srv.comp.task_package=\"hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://3.249.139.167:8000/rust-wasi-tutorial.zip\""
+    "golem.srv.comp.task_package=\"hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://yacn2.dev.golem.network:8000/rust-wasi-tutorial.zip\""
```
Comment (tworec) [inline core/market/resolver/tests/sample/mod.rs:116]:
```suggestion
    "golem.srv.comp.task_package=\"hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://12.34.56.78:8000/rust-wasi-tutorial.zip\""
```

**Pull request merged**